fixtures/starway/servo-color-4k.json

```diff
@@ -9,11 +9,8 @@
     "lastModifyDate": "2020-03-29"
   },
   "links": {
-    "manual": [
-      "https://en.star-way.com/telechargements/?doc=468441"
```

Comment:
There is a manual linked from the (new) product page: https://en.starway.eu/telechargements/?doc=474030

Reply:
Good spot! 👍